### ISSUE TITLE
Some questions about reproduce

### ISSUE THREAD

**Issue by GodKarna233** [kkoutini/PaSST#50]
Hi, I have successfully reproduced the model on the ESC50 and OpenMIC datasets. However, I have a question: the validation set used during the validation step in the training process seems to be the same as the test set used for the final evaluation of the model. Could this have any impact on the test results?

**Comment by GodKarna233** [comment]
Hi, I have set up the environment through the document and reproduced the result on ESC-50 via default config, but there are something wrong. When I excute python ex_esc50.py evaluate only after complete trainng, I dont know why the acc is just 2% or 3%.

**Comment by jd-rub** [comment]
Pinging @kkoutini to hopefully get an answer to this. 

I've noticed this as well and was also wondering where the hyperparameters like max_epochs come from (130 for AudioSet, 10 for OpenMIC and ESC50), given that there is no train/validation split with which to evaluate this choice independently.
